### ISSUE TITLE
verify-checksums does not list all added files

### ISSUE THREAD

**Issue by axos88** [wp-cli/checksum-command#67]
## Bug Report

- [x] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).
- [x] Yes, more specifically, I reviewed the guidelines on [how to write clear bug reports](https://make.wordpress.org/cli/handbook/bug-reports/).

**Describe the current, buggy behavior**

wp core verify-checksums (and I guess also for plugins) does not report all added files

**Describe how other contributors can replicate this bug**

```
# cd <path/to/wordpress>
# echo x > foo.php
# echo x > foo.txt
# sudo -u www-data wp core verify-checksums 
Warning: File should not exist: wp-salt.php
Success: WordPress installation verifies against checksums.
```

(Does not list foo.php and foo.txt)

**Describe what you would expect as the correct outcome**

The list should contain foo.php and foo.txt as "files should not exist"

**Let us know what environment you are running this on**

```
OS:	Linux webfe.akosv.com 3.13.0-147-generic #196-Ubuntu SMP Wed May 2 15:51:34 UTC 2018 x86_64
Shell:	/bin/bash
PHP binary:	/usr/bin/php5
PHP version:	5.5.9-1ubuntu4.29
php.ini used:	/etc/php5/cli/php.ini
WP-CLI root dir:	phar://wp-cli.phar/vendor/wp-cli/wp-cli
WP-CLI vendor dir:	phar://wp-cli.phar/vendor
WP_CLI phar path:	/var/www/wordpress/<snip>
WP-CLI packages dir:	
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	2.1.0
```

**Comment by axos88** [comment]
I don't think this should be closed. These files should be listed as warnings, and not arbitrarily silently ignored. At maximum a switch should enable these files to be ignored. Even better a .wpignore file could be added.

Our system was recently infected and trashed with all sorts of files. obviously not prefixed with wp-. I didn't have any normal means to detect and remove files from the root directory.

I, as a system administrator will know which files are installed by myself, and know to ignore them, but definately want to know if anything else was added without my knowledge. The current behaviour does not allow this check.

**Comment by axos88** [comment]
And you are exactly on point: wp-cli should not make a decision wether a file is legit or not. It should list everything that it sees, and let the user make the decision who actually has the necessary knowledge to make the decision. Arbitrarily filtering out files that do not start with `wp-` means wp-cli actually IS making that decision, and incorrectly.

**Comment by axos88** [comment]
Actually introducing the `.wp-ignore` file would give way to implementing `wp core verify-checksums --fix`.

**Comment by josenobile** [comment]
I strongly support the --fix flag, it saves time to properly format the files in order to put them as an argument in rm .

**Comment by marcinkrzeminski** [comment]
Hi,

I'm having kind of the same issue. While I understand that the root directory is specific and only prefixed files are verified, however I have created some file in `wp-content` and when I run the command they are not discovered by the `verify-checksums` command. 

Here's a screenshot (running on macOS Monterey 12.6, wp-cli at 2.6)

<img width="839" alt="CleanShot 2022-10-01 at 09 09 44@2x" src="https://user-images.githubusercontent.com/2551613/193397578-4cda163a-1520-413c-b46a-bfd1e26fee8c.png">
